corona-warn-app/cwa-app-android
# Antigen: PCR and Rapid test home screen cards (EXPOSUREAPP-6019, EXPOSUREAPP-6020, EXPOSUREAPP-6021)

PCR and Rapid test cards for Home Screen.

![Screenshot 2021-04-14 at 12 01 49](https://user-images.githubusercontent.com/64849422/114684161-52926080-9d19-11eb-951c-d05c2a8d2c07.png)


How to test:
1) test menu
2) "Home cards"
3) check all possible test cards for PCR and Rapid tests

Special note to testers:
Kindly check all these screens with different font size. Thank you 🚀
